src/server/index.tsx

```diff
@@ -34,7 +36,17 @@ server.listen(Number(port), hostname, () => {
   console.log(`http://${hostname}:${port}`);
 });
 
-process.on("SIGINT", () => {
+process.on("SIGINT", async () => {
+  const lang = I18NextService.i18n.language;
+  const locale = (
+    await import(
+      /* webpackExclude: /\.js\.flow$/ */
+      `date-fns/locale/${lang}`
+    )
+  ).default;
+  setDefaultOptions({
+    locale,
```

Comment:
Why here?

Reply:
I want to make sure the correct locale is used on both the server and the client.

Reply:
But this is the process hook for when the server shuts down, yeah?

Reply:
I am a fool and mistook this for where the server is started 🤦🏻

Reply:
Haha, no worries. It may be better to put this in a server util file too, like `init-date-fns.ts` and import it here instead.